ultralytics/nn/autobackend.py

```diff
@@ -95,6 +95,8 @@ class AutoBackend(nn.Module):
             | NCNN                  | *_ncnn_model/     |
             | IMX                   | *_imx_model/      |
             | RKNN                  | *_rknn_model/     |
+            | Triton Inference      | triton://model    |
+            | Executorch            | *.pte             |
```

Comment:
nit pick: our official name is ExecuTorch :)

Reply:
Thanks @mergennachin!

Reply:
All of them has been fixed. Thank you for catching.